src/main/kotlin/no/nav/familie/tilbake/oppgave/OppgaveService.kt

```diff
@@ -234,11 +236,21 @@ class OppgaveService(
             }
 
             else -> {
-                integrasjonerClient.ferdigstillOppgave(finnOppgaveResponse.oppgaver[0].id!!)
+                integrasjonerClient.ferdigstillOppgave(tilbakekrevingsOppgaver[0].id!!)
             }
         }
     }
 
+    private fun List<Oppgave>.filtrerTilbakekrevringsOppgave(): List<Oppgave> =
+        this.filter {
+            it.oppgavetype in
```

Comment:
Jeg tror it.oppgavetype er value. Se f.eks i KravvedtakstatusService eller Foreslåvedtakssteg hvordan oppgavetype brukes der
```
val behandleSakOppgavetyper = listOf(Oppgavetype.BehandleSak.value, Oppgavetype.BehandleUnderkjentVedtak.value)
            if (behandleSakOppgavetyper.contains(oppgave.oppgavetype)) {
            	// gjør noe
            }
	
```